cli/src/main/kotlin/tools/samt/cli/ASTPrinter.kt

```diff
@@ -21,7 +21,7 @@ class ASTPrinter private constructor() {
         }
 
         append(gray(" <${node.location}>"))
-        append(System.lineSeparator())
+        appendLine()
```

Comment:
Just FYI, `appendLine()` will always append a `\n` no matter what the OS is. But it seems to work in my Windows 11 environment so this is an improvement.